contracts/anchor_handler/lib.rs

```diff
@@ -207,10 +209,9 @@ mod anchor_handler {
             arguments: &[u8],
         ) -> Result<()> {
             if function_signature
-                == Keccak256::hash_with_four_bytes_output(
-                    b"set_handler([u8;32],[u8;8])".to_vec().as_slice(),
+                == blake2b_256_4_bytes_output(
+                    b"VAnchor::set_handler([u8;32],[u8;8])".to_vec().as_slice(),
```

Comment:
It is just the hash of the function name not the function signature.

Reply:
Oh...I missed this. The rest are not like that.
I have fixed the anomaly now